packages/grafana-ui/src/components/Forms/commonStyles.ts

```diff
@@ -86,30 +87,29 @@ export const inputSizesPixels = (size: string) => {
   }
 };
 
-export const getPropertiesForButtonSize = (props: StyleProps) => {
-  const { hasText, hasIcon, size } = props;
-  const { spacing, typography, height } = props.theme;
+export function getPropertiesForButtonSize(size: ComponentSize, theme: GrafanaTheme) {
+  const { typography, height } = theme;
 
   switch (size) {
     case 'sm':
       return {
-        padding: `0 ${spacing.sm}`,
+        padding: 8,
```

Comment:
I think we can use `SPACING_BASE` from `defaults.ts` as the base here, so the padding stay consistent with the ones from the design system .

Reply:
added spacing.base to the theme type so I could use it here